### ISSUE TITLE
Create index.html with overview of versions and use cases

### ISSUE THREAD

**Issue by peterdesmet** [enram/bird-migration-flow-visualization#130]
This page will live at http://enram.github.io/bird-migration-flow-visualization/viz

**Comment by peterdesmet** [comment]
I have 2 versions. Bart and I prefer the second version. @hvangasteren @jshamoun preferences?

## Version 1: all versions/use cases treated equal:

![bird migration flow visualization](https://cloud.githubusercontent.com/assets/600993/8329994/846f9a38-1a7e-11e5-8482-f492228d236c.png)

## Version 2: highlighting use cases, downplaying older versions:

![bird migration flow visualization 2](https://cloud.githubusercontent.com/assets/600993/8329989/7d80b6da-1a7e-11e5-9603-84eea79036fb.png)

**Comment by jshamoun** [comment]
I definitely prefer 1!

**Comment by peterdesmet** [comment]
You don't think it might be confusing to new users?

**Comment by jshamoun** [comment]
Not really, 1 line explanation that this is a landing page for different visualizations, its certainly not as hard to figure out as when my bank changes its website.

**Comment by hvangasteren** [comment]
Number 1 has a big advantage when more visualisations are/will be added. Two more from TIMAMP for instance. I would prefer the newest viz at the top.

**Comment by bartaelterman** [comment]
As Peter already mentioned, I'm a fan of number 2. That's kind of a spoiler for this comment, but let me explain why.

1. I don't think we should bother users with old versions. If nobody would have referred to our visualization, we would probably have just deleted the old version and only the new one would have been available. Although we can now keep old versions online using a stable url (allowing us to keep the nature viz stable for instance), our main goal is to have users look at the newest version. If they really insist, they can go back to a previous version.
2. As new case studies or new visualizations (TIMAMP for instance) get added, option 1 could become a mess where users will have to look for the right visualization on the right case study with the right version.
3. A general user experience design principle says: "don't give your users too much choices. If you can make a choice for them, do it". This is one of those cases. We should make the choice for our users: the latest version is what they want. And if there is an exception where a user would really want to see an older version, he can with the links at the bottom, but those are not distracting for other users. Remember, these users should be exceptions. If half of our users want to go back to a previous version, there is something wrong with our latest version.

Two seconds after I read your comments, I came across this example of how Google does it. Notice the blue bar:

![screen shot 2015-06-24 at 17 42 07](https://cloud.githubusercontent.com/assets/2742677/8348689/4e41b7e6-1b16-11e5-9bef-41c27df94da6.png)

The user is just presented with the new version, and if he insists, he can go back ("but really why?" :-) ) If Google would've presented me all the different versions of their forum page first, and I would have to choose, I would be overwhelmed. Not knowing which version I was looking at yesterday and not knowing which one to choose now.

As for @hvangasteren comment, I see no reason why new visualizations can not be added to design 2. You'll get the same effect as in design 1 (several tiles referring to different visualizations) the only difference is, the tiles will display different visualizations, not different versions of the same visualization.

Having said all that, I would propose a possible consensus design:

We only show the tiles of the latest version (so if you have 2 case studies and 2 visualizations, you get 4 tiles). At the bottom, we show a header "Older versions" with subheaders "version 2", "version 1" etc. But if you click on these links, they expand and show again 4 tiles (2 case studies, 2 visualizations) but this time they all refer to an older version.
The main difference is, by default, these tiles are hidden to make sure our users main focus goes to the latest version.

**Comment by jshamoun** [comment]
I know that I might sound like I have no opinion of my own, but in my defense, I can say that I try to listen.  Bart your logic is very clear to me.  Yesterday, I had responded from the visual perspective, it just looked really nice (version 1). Indeed though I see how this will get cluttered and confusing with time. I also understand the point that we see each new version as an improvement so why continuously draw attention to old versions? Nevertheless, I think it's nice to see the evolution of the visualizations and the discussions (very nice addition), by keeping an option to go to them if you really want to. In time you may even decide its no longer needed, but for now, they shouldn't be dropped just yet. 

In short, you have convinced me that version 2 makes more sense.

By the way, I really like the google bar, which made me laugh in the current context.

**Comment by peterdesmet** [comment]
Thanks for all the feedback and thanks Bart for the clear explanation. Note that we do not plan to add TIMAMP visualizations to this page, only flow visualizations (i.e. part of this repository). Also, all of this is not set in stone: we can adapt if new versions/use cases are created.

**Comment by bartaelterman** [comment]
I think Google just knew I needed that screenshot now to make a point.
